src/FamilyHubs.ReferralUi.Ui/Filters/ApiExceptionFilterAttribute.cs

```diff
@@ -31,7 +31,7 @@ public override void OnException(ExceptionContext context)
     private void HandleException(ExceptionContext context)
     {
         Type type = context.Exception.GetType();
-        if (_exceptionHandlers.ContainsKey(type))
+        if(_exceptionHandlers.TryGetValue(type, out _))
```

Comment:
Just curious, is this something Sonar complained about?

Reply:
Yes